apps/docs/app/api/crawlers/route.ts

```diff
@@ -157,9 +157,7 @@ async function functionDetails(
   return fullDescription + parameters + examples
 }
 
-function mdxToHtml(markdownUnescaped: string): string {
-  const markdown = markdownUnescaped.replace(/(?<!\\)\{/g, '\\{').replace(/(?<!\\)\}/g, '\\}')
```

Comment:
This was a hack for some previous reason where we had Markdown files with braces that couldn't be removed. I can't remember why they couldn't be removed at the time, but I searched through the JS reference page and couldn't find those exceptions anymore, so we should be safe to remove this, since it causes a different problem with MDX parsing.